src/model/modeling/cobra_helpers.py

```diff
@@ -97,6 +103,17 @@ def find_metabolite(model, id, namespace, compartment='e'):
     MetaboliteNotFound
         If no metabolites are found for the given parameters.
     """
+    # Add compartment postfix to metabolites in the BiGG namespace.
+    if namespace == "bigg.metabolite":
+        for model_compartment in model.compartments.keys():
+            if id.endswith(f"_{model_compartment}"):
+                logger.warning(
```

Comment:
The original idea was to expect the id _without_ compartment suffix being passed, and this would warn when that rule was being broken.

Since we found out that cobrapy is not adding these suffixes, I'm thinking we should now simply check for the passed identifier first, then just retry with the compartment appended if there is no match.

Reply:
> Since we found out that cobrapy is not adding these suffixes, I'm thinking we should now simply check for the passed identifier first, then just retry with the compartment appended if there is no match.

Yes, that sounds like the right approach to me.